smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -97,6 +98,9 @@ class Program
         [Option("--no-deployment", CommandOptionType.NoValue, Description = "Don't deploy Edge Hub and temperature sensor modules")]
         public bool NoDeployment { get; } = false;
 
+        [Option("--no-verify-scenarios", CommandOptionType.NoValue, Description = "Don't verify scenarios. E.g: verify is temperature sensor sent data")]
```

Comment:
Done. but removed default, because the idea will be to add more verification in the future (including plugable verification). 

---
In reply to: [208328336](https://github.com/Azure/iotedge/pull/131#discussion_r208328336) [](ancestors = 208328336)

Reply:
There's still a discrepancy here. You have `--no-verify` in one place and `--no-verify-scenarios` in another. I recommend using `--no-verify`. Also, what do you mean when you say "removed default"?

Reply:
Arh. :( Missed some spots. Fixed. 

---
In reply to: [208346214](https://github.com/Azure/iotedge/pull/131#discussion_r208346214) [](ancestors = 208346214)